test/data/cr_things_filter_update.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+apiVersion: stable.nicolerenee.io/v1
+kind: Character
+metadata:
+  name: thing1
+spec:
+  Name: Thing 1
+  From: Cat in the Hat
+  By: Dr. Suess
```

Comment:
This is totally my fault...but this is spelled wrong. It's Dr. Seuss

Reply:
Fixed